bin/hdfcoinc/pycbc_page_sensitivity

```diff
@@ -234,6 +247,14 @@ for j in range(len(args.bins)-1):
         pylab.plot(x_values, reach, alpha=alpha, c='black')
         pylab.fill_between(x_values, reach - elow, reach + ehigh,
                            facecolor=c, edgecolor=c, alpha=alpha)
+        if label and args.hdf_out:
+            plotdict['data/%s' % label] = reach
+            plotdict['error/%s' % label] = ehigh
+
```

Comment:
You might want to store the lower error bound as well. It's not guaranteed to be symmetric.